data/wanderer/wanderer news.txt

```diff
@@ -0,0 +1,127 @@
+
+news "wanderer civilians, no dialog"
+	location
+		government "Wanderer"
+		not attributes "evacuation"
+	name
+		word
+			"Gardener"
+	message
+		word
+			"You see a Wanderer carefully place some seeds in the ground. They then spread some sort of light grey powder over them."
+			"From the spaceport you can see some buildings with plants growing on their sides. A Wanderer prunes and waters them in mid-air."
+			"While walking through the spaceport, you see a large greenhouse full of various plants. A Wanderer inside is tending to them."
+			
+	name
+		word
+			"Group of Wanderers"
+	message
+		word
+			"While walking through the spaceport, you see several Wanderers flying through the air. You duck to avoid them."
+			"A crowd of Wanderers has formed around an object. Upon closer inspection, it appears to be some sort of technological device."
+			"As you walk by a group of Wanderers, they begin to coo and chitter at you. Evidently they have not seen something like you before."
+			
+news "Wanderers and Korath"
+	to show
+		has "Wanderers: Sestor: Final: done"
+	location
+			government "Wanderer"
+			not attributes "evacuation"
+	name 
+		word
+			"Wanderer merchant captain"
+			"Wanderer captain"
+	message
+		word
+			"The Korath have [enormous, gigantic] ships that many of them live on their whole lives. I cannot [imagine, fathom] doing the same."
+			"Have you [seen, viewed] a Korath up close? They only have feathers on their neck. How [strange, odd]."
+			"The engines the Korath use have no regard for [effciency, subtlety]. But I must admit, they are much more powerful than ours."
+			"The Korath cannot pay [well, adequately] for their transport jobs, but they insist on paying [despite it, nonetheless].
+			"I am trying to [learn, understand] more words of the Korath language. It is a difficult [task, undertaking], but it is important to learn the language of [those we work with, our peers]."
+	name
+		word
+			"Wanderer linguist"
+			"Linguist"
+	message
+		word
+			"Many scientists are currently [working on, engineering] a translation device that will allow direct [communication, dialog] with the Korath in their own language. It is [exciting, satisfying] to be part of this effort.
```

Comment:
The wanderers took the time to learn Hai rather than do such a thing. This seems an incongruous entry.

Reply:
I would argue that developing a translation device for the Korath is a more urgent task than developing one for the Hai, because the Wanderers are going to be directly working with Korath for a long time, probably millennia. It's probably easier for them to learn Hai than build a translation device that will probably not be needed once the transition through the Eye is complete.

Reply:
Perhaps, but someone has to learn it before they can do that effectively anyway. Perhaps, given the early stage of things, the present focus should be on whether or not it is possible for a Wanderer to fully speak the Korath language, and be assessing the need or scope requirements for a translator? Just like, dial it back a step?

Reply:
Alright, that makes sense.